index.js

```diff
@@ -771,6 +774,11 @@
     return x instanceof Right && R.eqProps('value', x, this);
   });
 
+  //  Right#extend :: Either a b ~> (b -> b) -> Either a b
```

Comment:
The spec defines:
>  `w.extend(f)`

and then goes on to say:
> `f` must return a value of type `v`, for some variable `v` contained in `w`

so I think `b -> b` is correct.